x-pack/plugins/ml/public/application/components/data_grid/common.ts

```diff
@@ -250,10 +251,6 @@ export const useRenderCellValue = (
         return cellValue ? 'true' : 'false';
       }
 
-      if (typeof cellValue === 'object' && cellValue !== null) {
-        return JSON.stringify(cellValue);
-      }
-
```

Comment:
Can you explain why this is no longer necessary?

Reply:
Seems like this line is a duplicate of line 228 of this file so I removed it